src/app/components/organization-picker.component.html

```diff
@@ -0,0 +1,59 @@
+<div *ngIf="loaded" class="d-flex">
+  <div class="dropdown">
+    <button
+      class="dropdown-toggle d-flex align-items-center btn disable-caret p-0 border-0"
+      type="button"
+      id="pickerButton"
+      data-toggle="dropdown"
+      aria-haspopup="true"
+      aria-expanded="false"
+    >
+      <app-avatar [data]="activeOrganization.name" size="45" [circle]="true"></app-avatar>
+      <div class="org-name ml-3">
+        <span>{{ activeOrganization.name }}</span>
+        <small class="text-muted">{{ "organization" | i18n }}</small>
+      </div>
+      <div class="ml-3">
+        <i class="bwi bwi-angle-down" aria-hidden="true"></i>
+      </div>
+    </button>
+    <div class="dropdown-menu" aria-labelledby="pickerButton">
+      <a
+        *ngFor="let org of organizations"
+        class="dropdown-item"
+        [routerLink]="['/organizations', org.id]"
+      >
+        <i
+          class="bwi bwi-check mr-2"
+          [ngClass]="org.id === activeOrganizationId ? 'visible' : 'invisible'"
```

Comment:
Any reason we use visible here vs ngIf?

Reply:
`invisible` hides the icon but it still takes up room in the DOM, keeping everything nicely aligned with the actual icon size.

Reply:
Marking this as resolved because it'll be in a separate PR